cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -173,27 +160,24 @@ public static void after() {
 
     private CertAndKey generateCa(OpenSslCertManager certManager, CertificateAuthority certificateAuthority, String commonName)
             throws IOException, CertificateException, KeyStoreException, NoSuchAlgorithmException {
+        String clusterCaStorePassword = "123456";
+
         File clusterCaKeyFile = File.createTempFile("tls", "cluster-ca-key");
         File clusterCaCertFile = File.createTempFile("tls", "cluster-ca-cert");
         File clusterCaStoreFile = File.createTempFile("tls", "cluster-ca-store");
-        try {
-            Subject sbj = new Subject.Builder()
-                .withOrganizationName("io.strimzi")
-                .withCommonName(commonName).build();
-
-            certManager.generateSelfSignedCert(clusterCaKeyFile, clusterCaCertFile, sbj, ModelUtils.getCertificateValidity(certificateAuthority));
-            certManager.addCertToTrustStore(clusterCaCertFile, CA_CRT, clusterCaStoreFile, clusterCaStorePassword);
-            return new CertAndKey(
-                    Files.readAllBytes(clusterCaKeyFile.toPath()),
-                    Files.readAllBytes(clusterCaCertFile.toPath()),
-                    Files.readAllBytes(clusterCaStoreFile.toPath()),
-                    null,
-                    clusterCaStorePassword);
-        } finally {
-            clusterCaKeyFile.delete();
-            clusterCaCertFile.delete();
-            clusterCaStoreFile.delete();
-        }
+        Subject sbj = new Subject.Builder()
+            .withOrganizationName("io.strimzi")
+            .withCommonName(commonName).build();
+
+        certManager.generateSelfSignedCert(clusterCaKeyFile, clusterCaCertFile, sbj, ModelUtils.getCertificateValidity(certificateAuthority));
+
+        certManager.addCertToTrustStore(clusterCaCertFile, CA_CRT, clusterCaStoreFile, clusterCaStorePassword);
+        return new CertAndKey(
+                Files.readAllBytes(clusterCaKeyFile.toPath()),
+                Files.readAllBytes(clusterCaCertFile.toPath()),
+                Files.readAllBytes(clusterCaStoreFile.toPath()),
+                null,
+                clusterCaStorePassword);
     }
```

Comment:
Realised that these were temp files being created by JVM, so shouldn't need to delete them manually, IIRC. If I've got that wrong, I'll use @tombentley's suggestion of Files.delete() for the exception on failure instead of checking the return code.

Reply:
Cheers @tombentley, will explicitly delete :)